### DISCUSSION TITLE
5A399ED9-85CD-4C9E-B6D3-B16DAF87973F

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#34] [General]
# HD 23472

HD 23472 is a binary star system of a K-type main sequence star and a red dwarf about 127.46 light-years away in the constellation Reticulum. The primary star (component A) has five planets, three of which are denser than Earth while the other two are comparable to Europa's

https://celestia.mobi/addon?item=5A399ED9-85CD-4C9E-B6D3-B16DAF87973F

**Comment by exocritic5457** [comment]
Textures are good, super-Mercuries are cool, and this is another relief from the majority of section K. I am a tad curious though, any link to paper mentioning said signals?

**Reply by Karistus** [reply]
The paper that mentioned the signals is Barros et al. 2022, the same one that confirmed the inner three planets. they mention it in a section at the end discussing the possibility of finding new planet candidates around the host
